src/index.js

```diff
@@ -23,3 +23,15 @@ const vm = new Vue({
     }
   }
 })
+
+Vue.component('tech', {
+  props: ['tech'],
```

Comment:
I think we should do a prop validation, like it's shown in [here](https://vuejs.org/v2/guide/components-props.html#Prop-Validation)